installer/dev/tracelogging.h

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "pch.h"
+#include <wil/resource.h>
+#include <wil/result_macros.h>
+#include "..\..\dev\WindowsAppRuntime_Insights\WindowsAppRuntimeInsights.h"
```

Comment:
Avoid .. in #include. Specify a  filename or relative pathname (like the wil includes above) and then add the right location to the include search path in the *.vcxproj

Reply:
A different header will instead be used here.